cmd/influx/main.go

```diff
@@ -56,6 +56,8 @@ func main() {
 	fs.IntVar(&c.ImporterConfig.PPS, "pps", defaultPPS, "How many points per second the import will allow.  By default it is zero and will not throttle importing.")
 	fs.StringVar(&c.ImporterConfig.Path, "path", "", "path to the file to import")
 	fs.BoolVar(&c.ImporterConfig.Compressed, "compressed", false, "set to true if the import file is compressed")
+	fs.StringVar(&c.ImporterConfig.RetentionPolicy, "retention-policy", "", "Overrides the retention policy when importing a databse from file")
```

Comment:
I think that this, and the `destination-database` flag should say something like "writes the import data to the provided retention policy when using -import". `Overrides` seemed a little confusion to me as to whether it was modifying the import file or changing the location the imported data was written to.